src/swc/options.ts

```diff
@@ -269,6 +272,8 @@ export default function parserArgs(args: string[]) {
     includeDotfiles: !!opts.includeDotfiles,
     deleteDirOnStart: Boolean(opts.deleteDirOnStart),
     quiet: !!opts.quiet,
+    only: opts.only || [],
```

Comment:
Hi @kdy1 , are we sure that defaults for `only` and `ignore` are properly propagated?

We already face issues with Nest.js failures (renovate PR builds) complaining about `only.length === 0`, while `only` is undefined.

Could you please check?

Reply:
It's fixed by https://github.com/swc-project/cli/pull/279

Reply:
Nice! Thanks.